src/core/PagePracticesV2/PagePracticesV2.tsx

```diff
@@ -57,6 +57,9 @@ const IE11Warning = styled.div`
 `;
 
 const PagePracticesV2: React.FC = () => {
+  window.scrollTo({
+    top: 0,
+  });
```

Comment:
To ensure this only runs once, can you wrap this in a `useEffect()` hook?
```suggestion
  useEffect(() => {
    window.scrollTo({
      top: 0,
    })
  }, []);
```

Reply:
I'm not so sure that is actually the behavior we want ... I don't think this component will actually unmount/remount on navigation since it's above the router

Reply:
no I was wrong, it does actually work. i don't understand why but what else is new